ern-local-cli/src/commands/create-api-impl.js

```diff
@@ -80,21 +82,25 @@ exports.handler = async function ({
     nativeOnly = await promptPlatformSelection()
   }
 
-  await generateApiImpl({
-    apiDependencyPath: DependencyPath.fromString(api),
-    outputFolder,
-    nativeOnly,
-    forceGenerate: force,
-    reactNativeVersion,
-    paths: {
-      apiImplHull: path.join(platformPath, `ern-api-impl-gen/hull`),
-      pluginsDownloadFolder: PLUGIN_FOLDER,
-      workingFolder: WORKING_FOLDER,
-      outFolder: ''
-    }
-  }).then(() => {
-    log.info('Success!')
-  })
+  try {
+    await generateApiImpl({
+      apiDependencyPath: DependencyPath.fromString(api),
+      outputFolder,
+      nativeOnly,
+      forceGenerate: force,
+      reactNativeVersion,
+      paths: {
+        apiImplHull: path.join(platformPath, `ern-api-impl-gen/hull`),
+        pluginsDownloadFolder: PLUGIN_FOLDER,
+        workingFolder: WORKING_FOLDER,
+        outFolder: ''
+      }
+    }).then(() => {
```

Comment:
Do not use `.then` is ... no really proper to mix/match ES7 async/await constructs with old ES6 promises.
you can just do instead : 

```javascript
try {
  await generateApiImpl({....})
  log.info('Success!')
} catch (e) {
  log.error(...)
}
```